stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -65,8 +61,8 @@ public protocol DistributedActorSystem: Sendable {
   /// `system.resolve(id: addr1, as: Greeter.self)` MUST return a reference
   /// to the same actor.
   func assignID<Act>(_ actorType: Act.Type) -> ActorID
-      where Act: DistributedActor,
-            Act.ID == ActorID
+    where Act: DistributedActor,
```

Comment:
Do we want all these indentation changes or it's just broken format?

Reply:
Broken format, my bad!